Library/Formula/natalie.rb

```diff
@@ -0,0 +1,13 @@
+require "formula"
+
+class Natalie < Formula
+  homepage "https://github.com/krzyzanowskim/Natalie"
+  url "https://github.com/krzyzanowskim/Natalie/archive/0.1.tar.gz"
```

Comment:
A 16 day old project with one release may well be too new for Homebrew. It is popular though, and not a self-submission, which swings in its favour.